### ISSUE TITLE
Our Play description is too long

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#1688]
Char limit is 4000.  Some of our translations exceed this. Though special if language has special characters (something else than utf-8) this might not be quite right).

>   3524 fastlane/metadata/android/af/full_description.txt
>   2404 fastlane/metadata/android/az-AZ/full_description.txt
>   3436 fastlane/metadata/android/bg/full_description.txt
>   4925 fastlane/metadata/android/bn-BD/full_description.txt
>   2329 fastlane/metadata/android/cs-CZ/full_description.txt
>   3753 fastlane/metadata/android/de-DE/full_description.txt
>   3371 fastlane/metadata/android/en-US/full_description.txt
>   2971 fastlane/metadata/android/eo/full_description.txt
>   3997 fastlane/metadata/android/es-ES/full_description.txt
>   2259 fastlane/metadata/android/et/full_description.txt
>   4151 fastlane/metadata/android/fi-FI/full_description.txt
>   4214 fastlane/metadata/android/fr-FR/full_description.txt
>   8288 fastlane/metadata/android/hi-IN/full_description.txt
>   4280 fastlane/metadata/android/hu-HU/full_description.txt
>   3762 fastlane/metadata/android/it-IT/full_description.txt
>   1485 fastlane/metadata/android/iw-IL/full_description.txt
>   6455 fastlane/metadata/android/kk/full_description.txt
>   3726 fastlane/metadata/android/ko-KR/full_description.txt
>   4055 fastlane/metadata/android/lt/full_description.txt
>  11156 fastlane/metadata/android/my-MM/full_description.txt
>   2094 fastlane/metadata/android/nl-NL/full_description.txt
>   3361 fastlane/metadata/android/no-NO/full_description.txt
>   3495 fastlane/metadata/android/pl-PL/full_description.txt
>   4015 fastlane/metadata/android/pt-PT/full_description.txt
>   6884 fastlane/metadata/android/ru-RU/full_description.txt
>   2251 fastlane/metadata/android/sk/full_description.txt
>   3606 fastlane/metadata/android/sl/full_description.txt
>   3619 fastlane/metadata/android/sr/full_description.txt
>   2210 fastlane/metadata/android/sr@latin/full_description.txt
>    720 fastlane/metadata/android/tr-TR/full_description.txt
>   6470 fastlane/metadata/android/uk/full_description.txt
>   2463 fastlane/metadata/android/uz/full_description.txt
>   2696 fastlane/metadata/android/zh-CN/full_description.txt
>   2696 fastlane/metadata/android/zh-TW/full_description.txt

We need to remove something. Strings are translated in parts (sentences or sometimes small paragraphs), so probably means removing one sentence totally.

Here's link to current english play description:
https://github.com/AndBible/and-bible/blob/develop/fastlane/metadata/android/en-US/full_description.txt

**Comment by tuomas2** [comment]
> "The provenance of all documents is researched to ensure that all documents can be legally distributed or that permission has been granted by the copyright holder. That is also the reason this app does not include documents like ESV, NIV and NKJV."

This is long sentence that is perhaps less interesting to users, but kind of important. But maybe it's enough to say it on our homepage etc.

**Comment by tuomas2** [comment]
Other suggestions are welcome. With this change it fits.

**Comment by BryceSteuer** [comment]
It seems like a lot of the less positive reviews mainly gravitate around the fact that AB is missing NIV, ESV. So it might be important to at least say _something_ about it in description.

**Comment by tuomas2** [comment]
For 4.0 we can't edit text or add anything new, because of translations. We can only remove text. 

Though, we could make exception for English text perhaps.

**Comment by tuomas2** [comment]
In 4.0 there's explanation for NIV, ESV and Amplified and Message in Download Manager (search ESV for example)

**Comment by agrogers** [comment]
I agree that that sentence is not a selling point. But I also agree that it
is good for people to know about it. But if you have to remove something
then that is a good candidate.
